src/components/EntityInfo.vue

```diff
@@ -13,7 +13,16 @@
             </div>
 
             <!-- Description -->
-            <div v-if="businessId" id="business-subtitle">{{ businessDescription }}</div>
+            <div v-if="businessId">
+              <span id="business-subtitle">{{ businessDescription }}</span>
+              <v-chip
+                v-if="isAuthorizedToContinueOut"
+                id="authorized-to-continue-out-chip"
+                class="primary mt-n1 ml-3 pointer-events-none" small label text-color="white"
+              >
+                <strong>AUTHORIZED TO CONTINUE OUT</strong>
+              </v-chip>
+            </div>
```

Comment:
Looks like this:

![image](https://user-images.githubusercontent.com/10002889/217100526-4fd18659-fbd2-4000-a32e-71eb2e9e3619.png)